pkg/reconciler/labeler/v2/metasync.go

```diff
@@ -145,14 +147,16 @@ func clearMetaForNotListed(ns, routeName string, acc Accessor, names sets.String
 // element through the provided accessor.
 // A nil route name will cause the route to be de-referenced, and a non-nil route will cause
 // that route name to be attached to the element.
-func setRoutingMeta(acc Accessor, ns, name, routeName string, remove bool) error {
+func setRoutingMeta(ctx context.Context, acc Accessor, ns, name, routeName string, remove bool) error {
 	if mergePatch, err := acc.makeMetadataPatch(ns, name, routeName, remove); err != nil {
 		return err
 	} else if mergePatch != nil {
 		patch, err := json.Marshal(mergePatch)
 		if err != nil {
 			return err
 		}
+		logger := logging.FromContext(ctx)
+		logger.Infof("Labeler V2 applying patch to %q. patch: %q", name, mergePatch)
```

Comment:
Do we want this at info or debug?

Reply:
Not sure. These are infrequent since they should only run once when making something active, so I was thinking we'd just show them at info. It's pretty important in terms of verifying that this actually happens (currently the labeler doesn't really log at all).

How do we typically choose between debug and info level?

Reply:
Something we're expecting to debug in user cluster or an important thingy: Info
Something is useful while developing and debugging e2e debug
(my purview)

Reply:
Debug it is